docs/designers-developers/developers/themes/theme-json.md

```diff
@@ -299,42 +308,53 @@ These are the current color properties supported by blocks:
 | Columns | Yes | Yes | Yes | Yes |
 | Group | Yes | Yes | Yes | Yes |
 | Heading [1] | Yes | - | Yes | Yes |
+| List | Yes | Yes | - | Yes |
 | Media & text | Yes | Yes | Yes | Yes |
+| Navigation | Yes | - | - | Yes |
 | Paragraph | Yes | - | Yes | Yes |
 | Post Author | Yes | Yes | Yes | Yes |
 | Post Comments | Yes | Yes | Yes | Yes |
 | Post Comments Count | Yes | Yes | - | Yes |
 | Post Comments Form | Yes | Yes | Yes | Yes |
 | Post Date | Yes | Yes | - | Yes |
 | Post Excerpt | Yes | Yes | Yes | Yes |
+| Post Hierarchical Terms | Yes | Yes | Yes | Yes |
 | Post Tags | Yes | Yes | Yes | Yes |
 | Post Title | Yes | Yes | - | Yes |
 | Site Tagline | Yes | Yes | - | Yes |
 | Site Title | Yes | Yes | - | Yes |
+| Template Part | Yes | Yes | Yes | Yes |
 
 [1] The heading block represents 6 distinct HTML elements: H1-H6. It comes with selectors to target each individual element (ex: core/heading/h1 for H1, etc).
 
+#### Spacing Properties
+
+| Context | Padding |
+| --- | --- |
+| Cover | Yes |
+| Group | Yes |
+
 #### Typography Properties
 
 These are the current typography properties supported by blocks:
 
-| Context | Font Size | Line Height |
-| --- | --- | --- |
-| Global | Yes | Yes |
-| Columns | - | - |
-| Group | - | - |
-| Heading [1] | Yes | Yes |
-| Media & text | - | - |
-| Paragraph | Yes | Yes |
-| Post Author | Yes | Yes |
-| Post Comments | Yes | Yes |
-| Post Comments Count | Yes | Yes |
-| Post Comments Form | Yes | Yes |
-| Post Date | Yes | Yes |
-| Post Excerpt | Yes | Yes |
-| Post Tags | Yes | Yes |
-| Post Title | Yes | Yes |
-| Site Tagline | Yes | Yes |
-| Site Title | Yes | Yes |
+| Context | Font Family | Font Size | Font Style | Font Weight | Line Height | Text Decoration | Text Transform |
```

Comment:
I think the markup for this isn't correct — it's not rendering in the preview: 

![Screen Shot 2020-11-11 at 9 14 25 AM](https://user-images.githubusercontent.com/1202812/98822079-59314a80-23fe-11eb-9ecd-1f35eced8e19.png)

Reply:
Good catch, thanks! It's fixed as of https://github.com/WordPress/gutenberg/pull/26891/commits/843e2fad3490268dded4cac54746ed3606e8a680